ui/static/ui/css/mit-lore.css

```diff
@@ -451,3 +451,7 @@ ul.repos {
     padding: 0px;
     margin: 0px;
 }
+
```

Comment:
If we are toggling this in javascript we might as well just set this value there on initialization.